src/ConcreteOperations/convex_hull.jl

```diff
@@ -443,6 +443,11 @@ construct the convex hull of a set of ``n`` points in the plane in
 ``O(n \\log n)`` time. For further details see
 [Monotone chain](https://en.wikibooks.org/wiki/Algorithm_Implementation/Geometry/Convex_hull/Monotone_chain)
 """
+
+_zero_abs(x::Number) = x
+
+_zero_abs(x::AbstractFloat) = x == -0.0 ? zero(x) : x
+
```

Comment:
These need to be moved (because they now separate the docstring from the function below).